src/resources/js/script.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file This file contains all JavaScript specific to Schedule Day View.
```

Comment:
Not sure the `@file` tag is a real thing ... might be cleaner just to go straight into a high level description?

Reply:
Ryan did this file initially and I didn't touch its header... but he might have started with /wp-content/plugins/the-events-calendar/src/resources/js/tribe-events-ajax-day.js